tests/unit/seed/embed/test_bootstrap_link_via_app_data.py

```diff
@@ -125,6 +143,55 @@ def test_base_bootstrap_link_via_app_data_not_writable(tmp_path, current_fastest
     assert result
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Windows only applies R/O to files")
```

Comment:
Seems like we still should run the test, but mark read only the files themselves, and not the dir, not?

Reply:
marking files read-only (and not directories) only prevents appending not deleting / creating / replacing so it's not a useful test

the reason I had to make *both* directories and files read-only is making a root directory read-only doesn't prevent writes in subdirectories

Reply:
So you're saying that pre-commit CI will not run on Windows then?

Reply:
correct it won't

but it doesn't matter -- you could reproduce this same test on windows but it involves creating an admin user and a non-admin user and I don't think you want to do that in your testsuite :)

Reply:
additionally, without my updates the test right above this wasn't actually testing anything 😆

Reply:
Cool.